ethpm_cli/parser.py

```diff
@@ -152,6 +174,13 @@ def release_cmd(args: argparse.Namespace) -> None:
     type=str,
     help="Content addressed URI at which the manifest for released package is located.",
 )
+release_parser.add_argument(
```

Comment:
Once we start having the same flag for multiple commands - I like to move them to helper fns - under `Shared args` at the top of this file. With this addition, there are 3 commands with a `manifest_path` flag, so it'd be nice to move it to a helper function - and you might also have to add an argument for the help message - since it's different between the commands. 

```
def add_manifest_path_arg_to_parser(parser, help_msg):
```

```
add_manifest_path_arg_to_parser(release_parser, "Local path to target manifest used for release.")
```

Reply:
Perfect, can this PR also do the same for the other reused args I might come across?